docker/Dockerfile

```diff
@@ -1,13 +1,19 @@
-FROM ubuntu:xenial
+FROM debian:stretch
 
-RUN apt-get update \
+RUN sed -i 's/101/0/' /usr/sbin/policy-rc.d \
```

Comment:
Maybe add a comment describing why we need to replace `101` with `0`? It's a bit confusing just looking at it...

Reply:
Done.